### PR TITLE
Add logic for error handling of start and stop commands

### LINKED ISSUE #435: mesheryctl: Include logic for handling errors in `start` and `stop`
**Description**
There are a number of unhandled errors in `mesheryctl` that should be handled in a graceful way. `mesheryctl` source code is here - https://github.com/layer5io/meshery/tree/master/mesheryctl

**Expected Behavior**
All errchecks should pass.

```
mesheryctl/cmd/start.go:40:12:	os.Mkdir(mesheryFolder, 0777)
mesheryctl/cmd/start.go:79:72:	exec.Command("rundll32", "url.dll,FileProtocolHandler", url).Start()
mesheryctl/cmd/start.go:82:41:	exec.Command("xdg-open", url).Start()
mesheryctl/cmd/start.go:85:37:	exec.Command("open", url).Start()
mesheryctl/cmd/stop.go:33:12:	os.Mkdir(mesheryFolder, 0777)
```

**Reference**
https://github.com/layer5io/meshery/commit/e8b20ff18fc7a05b0e4818fbe5843a09c138e7d5/checks?check_suite_id=285379602

### PR DESCRIPTION
**Pull request by supra08**
@leecalcote please review.

Fixes #435

### CODE REVIEW

File: mesheryctl/cmd/stop.go [hunk 1]
```diff
@@ -30,7 +30,9 @@ var stopCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		log.Info("Stopping Meshery...")
 		if _, err := os.Stat(mesheryFolder); os.IsNotExist(err) {
-			_ = os.Mkdir(mesheryFolder, 0777)
+			if err := os.Mkdir(mesheryFolder, 0777); err != nil {
+				log.Fatal("[ERROR] Could not create Meshery directory. The error message: \n", err)
```
Comment (anandrkskd) [inline mesheryctl/cmd/stop.go]: Same here!

File: mesheryctl/cmd/start.go [hunk 2]
```diff
@@ -71,18 +73,27 @@ var startCmd = &cobra.Command{
 		//check for container meshery_meshery_1 running status
 		for _, container := range containers {
 			if "/meshery_meshery_1" == container.Names[0] {
-				log.Info("Opening Meshery in your broswer. If Meshery does not open, please point your browser to http://localhost:9081 to access Meshery.")
+				log.Info("Opening Meshery in your browser. If Meshery does not open, please point your browser to http://localhost:9081 to access Meshery.")
```
Comment (anandrkskd) [inline mesheryctl/cmd/start.go]: Thanks for pointing this out!

File: mesheryctl/cmd/start.go [hunk 3]
```diff
@@ -37,7 +37,9 @@ var startCmd = &cobra.Command{
 		// var stderr bytes.Buffer
 
 		if _, err := os.Stat(mesheryFolder); os.IsNotExist(err) {
-			_ = os.Mkdir(mesheryFolder, 0777)
+			if err := os.Mkdir(mesheryFolder, 0777); err != nil {
+				log.Fatal("[ERROR] Could not create Meshery directory. The error message: \n", err)
```
Comment (anandrkskd) [inline mesheryctl/cmd/start.go]: Hey @supra08 the changes looks great, 
But there is some slight changes that are required  we are not required to print  "[ERROR]" in statements. log.fatal will itself do the work!
you just need to remove "[ERROR]" form statements.

**Comment by leecalcote** [comment]
Good work, @supra08 and @anandrkskd.

**Comment by supra08** [comment]
@anandrkskd made the changes.

**Pull request merged**